.changes/next-release/enhancement-Argumentprocessing-46505.json

```diff
@@ -0,0 +1,5 @@
+{
+  "category": "Argument processing",
+  "type": "enhancement",
+  "description": "Added cli_follow_urlparam option in the config file which can be set to false to disable following of http"
```

Comment:
Maybe we should update this to be "to disable following of parameter values that start with `http://` or `https://`"? Right now it is a little unclear of when it get follows.